vscode-extension/media/faq.md

```diff
@@ -0,0 +1,40 @@
+# FAQ / Troubleshooting
+
+Whenever you encounter errors, they should show up in the Output channel. Access it by following these steps:
+
+1.  Press keys "CMD" + "Shift" + P
+2.  Type and enter "Output: Show Output Channels"
+3.  Type and enter "AIConfig"
+
+**Q: I'm seeing a blank page when I run "AIConfig: Create New"**  
+A: This likely means either:
+
+1.  Missing or outdated python modules
+    - Make sure you're on version >= 0.0.6 for AIConfig Editor extension
+    - Type `pip3 list | grep aiconfig` in terminal and make sure you have `python-aiconfig` >= 1.1.22. If it's not on >=1.1.22, run `pip3 install python-aiconfig --force-reinstall`
+    - Check the output channel for more info (see instructions at beginning of "FAQ / Troubleshooting")
+2.  You are running AIConfig Editor with the wrong Python interpreter
+    - Make sure you have Python >=3.10 installed
+    - After you try to create a new file, and viewing blank screen, open command palette ("CMD" + "SHIFT" + P) and run the command `AIConfig: Initialize Extension` to ensure everything is properly configured
+
+**Q: Got this error: "Failed to start aiconfig server. You can view the aiconfig but cannot modify it."**  
+A: This can happen when trying to open a config file. You can usually fix this by:
+
+1.  Re-opening the file
+    - If you still see this error but the file opens and works, you can ignore this for now
+2.  Running the steps to make sure your Python pacakges are properly initialized and up-to-date
+    - Follow instructions for "I'm seeing a blank page" issue from above
```

Comment:
I have only ever seen this error when opening a file which has invalid config (i.e. model parsers that are not installed -- though maybe we can split this into 2 categories, one for truly invalid config that is malformed or doesn't match config spec, and one for missing extension dependencies?). In that case, the Output channel clearly specifies the error. We should add this here, either as a 3. to the blank page section or as another step here.